lib/jormungandr/exe/cardano-wallet-jormungandr.hs

```diff
@@ -412,3 +415,39 @@ helperTracingText = unlines $
         "  " ++ padRight maxLength ' ' name ++ "  " ++ desc
       where
         padRight n char str = take n $ str ++ replicate n char
+
+{-------------------------------------------------------------------------------
+                                    Logging
+-------------------------------------------------------------------------------}
+
+data Log
```

Comment:
we use `ApplicationLog` in ` lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs` - maybe we could give some prefix to `Log` ?

Reply:
I changed it to `MainLog`.